var/spack/repos/builtin/packages/cbtf-krell/package.py

```diff
@@ -262,3 +262,10 @@ def setup_environment(self, spack_env, run_env):
 
         run_env.set('CBTF_MRNET_BACKEND_PATH',
                     self.prefix.sbin.cbtf_libcbtf_mrnet_backend)
+
+        run_env.prepend_path('PATH', self.spec['libmonitor'].prefix.bin)
+        run_env.prepend_path('PATH', self.spec['papi'].prefix.bin)
+        run_env.prepend_path('PATH', self.spec['mrnet'].prefix.bin)
+        mpath = '/share/man'
+        run_env.prepend_path('MANPATH', self.prefix + mpath)
+        run_env.prepend_path('MANPATH', self.spec['cbtf-krell'].prefix + mpath)
```

Comment:
This should already be set in the default `modules.yaml` configuration.

Reply:
Yes, the module paths are there.  I took the lines related to module paths out.